frontend/src/modules/Datasets/views/DatasetCreateForm.js

```diff
@@ -29,15 +29,15 @@ import {
   ChipInput,
   Defaults,
   useSettings
-} from '../../../../design';
-import { SET_ERROR, useDispatch } from '../../../../globalErrors';
+} from '../../../design';
```

Comment:
Datasets (and everything related to it as tables, locations/folders, shares) is one of the optional modules. The views should not be shown if the module is disabled. We should do the same as it has been done for notebooks

Reply:
can you please elaborate?

Reply:
We shouldn't show dataset's forms if the dataset module is inactive. It was one of the requirements of the modularization: s customer can specify what modules are active/inactive in the config.json. 
The other example of the module that we can make inactive is `notebook`. If we set `modules.notebook.active = false` in the config.json, we won't see anything related to notebooks. For the reference, please take a look at #415. 
The list of the optional [modules](https://github.com/awslabs/aws-dataall/blob/modularization-main/config.json)